src/vs/workbench/contrib/notebook/browser/contrib/find/notebookFindReplaceWidget.ts

```diff
@@ -727,4 +789,21 @@ registerThemingParticipant((theme, collector) => {
 	if (inputActiveOptionBackgroundColor) {
 		collector.addRule(`.simple-fr-find-part .find-filter-button > .monaco-action-bar .action-label.notebook-filters.checked { background-color: ${inputActiveOptionBackgroundColor}; }`);
 	}
+
+	const resizeBorderBackground = theme.getColor(editorWidgetResizeBorder);
+	if (resizeBorderBackground) {
+		collector.addRule(`.monaco-workbench .simple-fr-find-part-wrapper .monaco-sash { background-color: ${resizeBorderBackground}; }`);
+	} else {
+		const border = theme.getColor(editorWidgetBorder);
+		if (border) {
+			collector.addRule(`.monaco-workbench .simple-fr-find-part-wrapper .monaco-sash { background-color: ${border}; }`);
+		}
+	}
+
+	collector.addRule(`
+	:root {
+		--notebook-find-width: ${NOTEBOOK_FIND_WIDGET_INITIAL_WIDTH}px;
+		--notebook-find-horizontal-padding: ${NOTEBOOK_FIND_WIDGET_INITIAL_HORIZONTAL_PADDING}px;
```

Comment:
setting these traits dynamically so that it can use the constants in this file (which, in turn, is used in width calculations)